DQM/L1TMonitor/interface/L1TStage2uGT.h

```diff
@@ -75,9 +75,16 @@ class L1TStage2uGT: public DQMEDAnalyzer {
    int algoBitLastBxInTrain_;
    const std::string algoNameFirstBxInTrain_;
    const std::string algoNameLastBxInTrain_;
-   
+ 
+   int algoBitUnpre_;
+   int algoBitDis_; 
+   std::vector<std::string> unprescaledAlgoShortList_ ;
+   std::vector<std::string> disabledAlgoShortList_; 
```

Comment:
Better call the variables prescaledAlgo*** instead of disabledAlgo*** since not all algos in the list will be disabled.